support-frontend/assets/helpers/subscriptionsForms/formActions.ts

```diff
@@ -167,7 +167,7 @@ const setUserTypeFromIdentityResponse = (
 const formActionCreators = {
 	setTitle: (title: string): Action => ({
 		type: 'SET_TITLE',
-		title: title !== '' ? title : null,
+		title: title !== 'Select a title' ? title : null,
```

Comment:
Is there never a point where we need to check for an empty string?

Reply:
I don't think there would be, as the user can only select their title from the predetermined list in the dropdown and on the initial load the value is set to null. Although as a failsafe I can modify this to check for the string above as well as an empty string?

Reply:
No that's fine, just wanted to check!